lib/inject.js

```diff
@@ -38,11 +38,12 @@ function findFramesAndInject(parent, script, driver) {
 /**
  * Get the source of the axe-core package
  * @private
+ * @param {String}	axeSource Optional custom source and rules passed in through constructor
  * @param {Function} callback Callback to execute when source is retrieved
  */
-function getSource(callback) {
-	if (AXE_SOURCE) {
-		return callback(null, AXE_SOURCE);
+function getSource(axeSource, callback) {
+	if (axeSource !== null) {
+		return callback(null, axeSource);
 	}
 	var pathToAxe = path.resolve(__dirname, '../node_modules/axe-core/axe.js');
```

Comment:
That is most of the point of this PR–to allow the source to be passed in as a parameter, or alternatively, look for it in the node_modules.